.github/workflows/azure_ci_check.yml

```diff
@@ -0,0 +1,72 @@
+name: Azure CI
```

Comment:
does the .yml file have a apache license we can add?

Reply:
Existing .yml files don't have Apache license and .yml files are not checked for license.  We can still add that as a comment at the beginning.

Reply:
Apache license added.